index.bs

```diff
@@ -703,6 +703,21 @@ In the <a>HTML Processing Model</a>, under the "<i>Update the rendering</i>" ste
 
 <ul>The |document| does not have <a>pending initial IntersectionObserver targets</a>.</ul>
 
+<!-- ============================================================ -->
+<h2 id=accessibility>Accessibility Considerations</h2>
+<!-- ============================================================ -->
+
+*This section is non-normative.*
+
+There are no known accessibility considerations for the core IntersectionObserver specification (this document).
+There are, however, related specifications and proposals that leverage and refer to this spec,
+which might have their own accessibility considerations. In particular, specifications for
+the <a href="https://github.com/whatwg/html/issues/2806">lazy loading of images</a> and
+<a href="https://github.com/WICG/display-locking">display locking</a> may have implications for
+<a href="https://html.spec.whatwg.org/multipage/interaction.html#find-in-page">find-in-page</a>,
+<a href="https://wicg.github.io/spatial-navigation/">spatial navigation</a>,
+and <a href="https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute">tab index</a> features.
```

Comment:
The text seems fine, but (nit:) please avoid linking to HTML directly (to avoid breakage if things get moved/renamed). Please use `[=tabIndex=]` or whatever the appropriate BikeShed linking mechanism is. @Emilio should be able to recommend the right solution here. Same with the other specs - please don't link to those directly.

Reply:
Also, if you think appropriate, don't be shy about asking the HTML editors to export additional definitions if needed.